### PR TITLE
AC-7856 - Replace instances of AcceleratorConfig.name with a constant

### PR DESCRIPTION
**Pull request by pbx**
## [AC-7856](https://masschallenge.atlassian.net/browse/AC-7856)

Review suggestions:
* Note that [Travis build](https://travis-ci.com/github/masschallenge/django-accelerator/builds/197436197) is green
* Note that [Travis build of same-named accelerate branch](https://travis-ci.com/github/masschallenge/accelerate/builds/197371041) (which has no changes, but causes this branch of DA to be used in the build) is green

### CODE REVIEW

**Comment by pbx** [comment]
**Note:** even though most New Monolith PRs to date have only needed one reviewer, **this PR needs our standard two reviewers** because it is merging to `development`, not a long-running branch.

File: accelerator/models/profile_query_set.py [hunk 1]
```diff
@@ -14,18 +11,13 @@
...
-ExpertProfile = swapper.load_model(AcceleratorConfig.name,
-                                   "ExpertProfile")
-MemberProfile = swapper.load_model(AcceleratorConfig.name,
-                                   "MemberProfile")
-BaseProfile = swapper.load_model(AcceleratorConfig.name,
-                                 "BaseProfile")
+EntrepreneurProfile = swapper.load_model('accelerator', "EntrepreneurProfile")
+ExpertProfile = swapper.load_model('accelerator', "ExpertProfile")
+MemberProfile = swapper.load_model('accelerator', "MemberProfile")
+BaseProfile = swapper.load_model('accelerator', "BaseProfile")
```
Comment (iankigen) [inline accelerator/models/profile_query_set.py:20]: why not have `accelerator` as a constant in `settings.py`?

Comment (pbx) [reply accelerator/models/profile_query_set.py:20]: That would be reasonable. However, this code is not going to be around very long - in Stage 3 (which begins in a week or two) we'll be doing [AC-8324](https://masschallenge.atlassian.net/browse/AC-8324)"Remove model-swapping code from django-accelerator and accelerate"

**Review by iankigen — APPROVED** [review]
Looks good
- Instances of `AcceleratorConfig.name` are replaced with `accelerator` ✅ 
- Travis build ✅

**Review by geofrocker — APPROVED** [review]
QA passes

**Pull request merged**